smst.go

```diff
@@ -71,27 +77,38 @@ func (smst *SMST) Spec() *TrieSpec {
 	return &smst.TrieSpec
 }
 
-// Get retrieves the value digest for the given key and the digest of the value
-// along with its weight provided a leaf node exists.
+// Get retrieves the value digest for the given key, along with its weight assuming the node exists.
```

Comment:
```suggestion
// Get retrieves the value digest for the given key, along with its weight assuming
// the node exists, otherwise the default placeholder values are returned 
```